### ISSUE TITLE
[Explore] Is there an easier way for users to install Korifi on kind

### ISSUE THREAD

**Issue by danail-branekov** [cloudfoundry/korifi#2696]
Our install on kind instructions often become outdated and is hard for people to follow. In order to address that, could we
a) come up with a script that we release in the release artifacts that installs on kind? Maybe pick up bits from korifi-ci that install already released versions
b) or even better, can we release a docker image that contains a kind cluster with korifi deployed?
c) explore any better alternatives

Whatever we do, we should test the installer in our CI pipeline

**Comment by sylvainkalache** [comment]
I recently started to use Korifi, so I am happy to share my experience.

Overall I like being able to streamline the installation of dependencies and Korifi with the `deploy-on-kind.sh` script. When someone's goal isn't to run a production environment, this method is perfect.

Some issues I faced with this script:
1) kpack timing out. I did not keep track of all the instances when this happened, but there is a 10s timeout that I have often faced, for example: 
`Internal error occurred: failed calling webhook "defaults.webhook.kpack.io": failed to call webhook: Post "[https://kpack-webhook.kpack.svc:443/defaults?timeout=10s](https://kpack-webhook.kpack.svc/defaults?timeout=10s)": context deadline exceeded`

2) script failing with obscure error messages. It turns out Korifi requires specific versions of the prerequisites; it's not documented, so I could not know. My system package manager was installing older versions than what was needed; it took me some time to figure this out. 

3) In my mind, when running the `deploy-on-kind.sh` script, I thought it would install everything it needed to work. And since I did not RTFM (shame on me), I missed the prerequisite section :-) One idea could that `deploy-on-kind.sh` also installs the prerequisite as part of the `deploy-on-kind.sh` script.

Another alternative would be to prompt the user, when running `deploy-on-kind.sh`, whether they want the script to install the prerequisites. 

Another alternative could be to provide the script that installs the prerequisites as a standalone one.

I submitted an issue about this specific topic a few days ago and already wrote a script that installs prerequisites. So happy to further help on that topic
https://github.com/cloudfoundry/korifi/issues/2742

I absolutely agree with the "test the installer in our CI pipeline". Installing Korifi manually is, at this point, quite a lot of work, and having a smooth onboarding experience for newcomers is essential.

Ultimately I think the community should aim to package Korifi so that one can just apt-get install, but I guess that's for later ;-)
